app/namespace.go

```diff
@@ -429,6 +430,11 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 					Usage:   fmt.Sprintf("Flag can be used multiple times; value must be \"email=permission\"; valid permissions are: %v", getNamespacePermissionTypes()),
 					Aliases: []string{"p"},
 				},
+				&cli.StringFlag{
+					Name:    codecServerSpecJsonFlagName,
+					Usage:   `JSON that defines the codec server property passed to the namespace. Sample JSON: { "codecserver": { "endPoint": "https://test-endpoint.com", "PassAccessToken": true, "IncludeCredentials": false } }`,
```

Comment:
can we run this by jonathan/abhinav? I think from a user experience its easier to just take three separate flags than to require the user to construct the JSON themselves

Reply:
Changed to use separate flags.